app/models/item.js

```diff
@@ -40,22 +40,43 @@ export default cloudinaryUrl.extend({
     async: false
   }),
   designationId: attr("string"),
-  designation: belongsTo("designation", { async: true }),
-  location: belongsTo("location", { async: false }),
-  code: belongsTo("code", { async: false }),
-  packageType: belongsTo("packageType", { async: false }),
-  donorCondition: belongsTo("donor_condition", { async: false }),
-  setItem: belongsTo("set_item", { async: false }),
-  packagesLocations: hasMany("packages_location", { async: true }),
+  designation: belongsTo("designation", {
+    async: true
+  }),
+  location: belongsTo("location", {
+    async: false
+  }),
+  code: belongsTo("code", {
+    async: false
+  }),
+  packageType: belongsTo("packageType", {
+    async: false
+  }),
+  donorCondition: belongsTo("donor_condition", {
+    async: false
+  }),
+  setItem: belongsTo("set_item", {
+    async: false
+  }),
+  packagesLocations: hasMany("packages_location", {
+    async: true
+  }),
 
   storageTypeId: attr("number"),
   storageType: belongsTo("storage_type", {
     async: true
   }),
 
-  ordersPackages: hasMany("ordersPackages", { async: true }),
+  addedQuantity: attr("number"),
+  inHandQuantity: attr("number"),
```

Comment:
If you could rename to `onHandQuantity`, both to be more correct and to match the upcoming counter caches column that we'll be adding : )